code/components/citizen-resources-client/src/ConsoleScriptFunctions.cpp

```diff
@@ -12,8 +12,9 @@ static InitFunction initFunction([]()
 
 		// this should not be exposed to script
 		// #TODO: 'not exposed to script' convar flags
-		if (HashString(varName.c_str()) == HashString("cl_ownershipTicket") || // ownership ticket is *very* bad to expose as users can pretend to be other users using this
-			HashString(varName.c_str()) == HashString("ui_updateChannel"))     // update channel is often misused as a marker for other things
+		if (HashString(varName.c_str()) == HashString("cl_ownershipTicket") ||   // ownership ticket is *very* bad to expose as users can pretend to be other users using this
+			HashString(varName.c_str()) == HashString("ui_updateChannel") ||     // update channel is often misused as a marker for other things
+			HashString(varName.c_str()) == HashString("cl_discordRichPresence")) // avoid scripts denying user-choice
```

Comment:
Weird all these line changes, though while you're at it, can you cache `HashString(varName.c_str())` and check that variable against all these others? And I think we should go with a more constructive comment like "Users probably set this for privacy reasons, no need for peeking eyes"

Reply:
Line changes were due to comment indentation.